docs/pages/eas-update/estimate-bandwidth.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Estimate bandwidth usage
+description: Learn how to estimate bandwidth usage with EAS Update.
+---
+
+# Understanding update bandwidth usage
+
+EAS Update enables an app to update its own non-native pieces (such as JS, styling, and images) over-the-air. This guide explains how bandwidth is consumed and optimized.
+
+## Bandwidth calculation breakdown
+
+Each subscription plan includes a predefined bandwidth allocation per billing cycle, which translates to a specific number of updates your users can download. Here's how we estimate bandwidth usage:
```

Comment:
Each plan also includes an additional amount of bandwidth per MAU I think. From the pricing page:

> 1 TiB of global edge bandwidth included. For every additional MAU over 50,000 MAUs, 40 MiB of bandwidth is added to your bandwidth quota. After that, you'll be charged $0.10 per GiB.

So maybe this should be worded like:


```suggestion
Each subscription plan includes a predefined bandwidth allocation per billing cycle plus 40 MiB per MAU for non-on-demand plans. Here's how to estimate bandwidth usage per update:
```